miro_preprocessor/s3.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_s3_bucket" "wellcomecollection-images" {
+  bucket = "wellcomecollection-images"
```

Comment:
Is this bucket storing the master assets? If so, is it appropriate that it lives in the Miro preprocessor stack?